openedx/core/djangoapps/oauth_dispatch/docs/decisions/0008-use-asymmetric-jwts.rst

```diff
@@ -89,62 +89,78 @@ KeyPair Generation
 
 Here is code for generating a keypair::
 
+    import json
```

Comment:
I wonder if rather than having all of this example code to maintain (and that isn't covered by tests) we should just point to the places we actually use the code. There's a management command for generating and serializing keypairs, and there's code in edx-platform and edx-drf-extensions for signing and validating.

My only concerns with that would be 1) code can move over time, so maybe we'd want to use permalinks to code at specific commits, and 2) some of the code is overly complicated because it still has branches for symmetric signing.

@robrap Any thoughts on this? It's the second time recently I've run into example code in ADRs.

Reply:
I agree that ADRs might include pseudo-code to add clarity, but shouldn't have actual code. I am only responding to @timmc-edx's comment, and I am not looking at this code.